janus.c

```diff
@@ -3680,6 +3667,7 @@ gint main(int argc, char *argv[])
 		gboolean private_address = FALSE;
 		const char *test_ip = nat_1_1_mapping ? nat_1_1_mapping : local_ip;
 		struct sockaddr_in addr;
+		/* FIXME This ugly check only works with IPv4... */
```

Comment:
Thanks, I'll integrate this: we do indeed have some redundant methods in `utils.c` that should be removed/replaced, and this seem to do the trick (besides better handling the scenario you referred to).